pkg/multicluster/razee/razee_cluster_manager.go

```diff
@@ -193,6 +194,19 @@ func (r *ClusterManager) CreateBlueprint(cluster string, blueprint *v1alpha1.Blu
 		return err
 	}
 
+	// Check if channel exists
+	existingChannel, err := r.con.Channels.ChannelByName(r.orgId, channelName)
+	if err != nil {
+		if !strings.HasPrefix(err.Error(), "Query channelByName error. Could not find the channel with name") {
```

Comment:
We should check if Razee exposed error codes instead and if so open an issue in satcon to expose it. For now, maybe set this as a constant?

Reply:
I checked. Not found is actually not a 404 but a 200 with an error code. (returned from razee)